src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private String name = "circle";
```

Comment:
Looks like a constant. You are using it in the `toString` method.
But I think you can remove it, and just type that word in a string literal in `toString` method.